server/routes/induction/update/workedBeforeUpdateController.ts

```diff
@@ -0,0 +1,74 @@
+import { NextFunction, Request, RequestHandler, Response } from 'express'
+import createError from 'http-errors'
+import type { InductionDto } from 'inductionDto'
+import type { WorkedBeforeForm } from 'inductionForms'
+import WorkExperienceController from '../common/workedBeforeController'
+import toCreateOrUpdateInductionDto from '../../../data/mappers/createOrUpdateInductionDtoMapper'
+import logger from '../../../../logger'
+import { InductionService } from '../../../services'
+import validateWorkedBeforeForm from './workedBeforeFormValidator'
+import YesNoValue from '../../../enums/yesNoValue'
+
+/**
+ * Controller for the Update of the WorkExperience screen of the Induction.
+ */
+export default class WorkedBeforeUpdateController extends WorkExperienceController {
+  constructor(private readonly inductionService: InductionService) {
+    super()
+  }
+
+  getBackLinkUrl(req: Request): string {
+    const { prisonNumber } = req.params
+    return `/plan/${prisonNumber}/view/work-and-interests`
+  }
+
+  getBackLinkAriaText(_req: Request): string {
+    return 'Back to <TODO - check what CIAG UI does here>'
```

Comment:
👍 
We'll fix all these up towards the end of all the work we are doing re: Updates 👍